locales/en/manage-kafka-permissions.json

```diff
@@ -11,6 +11,7 @@
   "dialog_aria_label": "Manage access dialog",
   "dialog_title": "Manage access",
   "is_kafka_instance": "is '{{value}}' ",
+  "is_kafka_instane": "",
```

Comment:
Did you mean to include this new line? It looks like it was unintentional.

Reply:
No I don't thinks so, I'll take that out.